src/App/psr.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+namespace App;
+
+/**
+ * File doc
+ * PHP version 7.4
+ *
+ * @category Category
+ * @package  Package
+ * @author   Octavio <octavio.marchi@gmail.com>
+ * @license  http://www.opensource.org/licenses/mit-license.html  MIT License
+ * @link     5
+ */
+
+/**
+ * Class doc
+ *
+ * @category Category
+ * @package  Package
+ * @author   Octavio <octavio.marchi@gmail.com>
+ * @license  http://www.opensource.org/licenses/mit-license.html  MIT License
+ * @link     5
+ */
+class psr
```

Comment:
:no_entry_sign: **Error**: Class name &quot;psr&quot; is not in PascalCase format (*Squiz.Classes.ValidClassName.NotCamelCaps*).